test/example.rb

```diff
@@ -2,6 +2,8 @@ def greeting(name)
   puts "Hello, #{name}! Welocome to the Ruby typos test."
   puts 'languege' # typo
   puts 'knowlege' # typo
+  puts 'knowlege: languege' # typo
+  puts 'welcom'
 end
```

Comment:
```suggestion
  puts 'welcome'
```
Should replace `welcom` with `welcome`.